.github/workflows/test-application.yaml

```diff
@@ -5,28 +5,50 @@ on:
     push:
         branches:
             - master
+            - develop
             - '[0-9]+.[0-9]+'
 
 jobs:
     test:
         name: "Run tests with php ${{ matrix.php-version }}"
-        runs-on: ubuntu-18.04
+        runs-on: ubuntu-latest
+
+        env:
+            DATABASE_URL: mysql://root:root@127.0.0.1:3306/su_theme_test?serverVersion=5.7
+            DATABASE_CHARSET: utf8mb4
+            DATABASE_COLLATE: utf8mb4_unicode_ci
 
         strategy:
             fail-fast: false
             matrix:
                 include:
-                    - php-version: '5.5'
-                      composer-flags: '--prefer-dist --prefer-lowest --no-interaction --prefer-stable'
+                    - php-version: '7.3'
+                      database: mysql
+                      phpcr-transport: doctrinedbal
+                      composer-flags: '--prefer-lowest --prefer-stable --prefer-dist --no-interaction'
                       tools: 'composer:v1'
+                      lint: false
                       env:
-                          SYMFONY__DEPRECATIONS__HELPER: disabled
+                          SYMFONY_DEPRECATIONS_HELPER: disabled
 
                     - php-version: '7.4'
+                      database: mysql
+                      phpcr-transport: doctrinedbal
                       composer-flags: '--prefer-dist --no-interaction'
                       tools: 'composer:v2'
+                      lint: true
+                      php-extensions: 'imagick'
```

Comment:
all matrix variables should be defined on each job. So also define this in the `php7.3` job